src/structbilty.py

```diff
@@ -640,23 +659,36 @@ def get_features(self, seqs, train=False, update=True, embeds_in_file=False):
         """
         # word embeddings
         words = seqs.words
-        embeds = seqs.embeds
-        wfeatures = np.array([self.get_w_repr(word, train=train, update=update, embeds_in_file=embeds_in_file, embed=embed) for word, embed in zip(words, embeds)])
-        
+        wfeatures = np.array(
+            [self.get_w_repr(word, train=train, update=update) for word in words])
+
         lex_features = []
         if self.dictionary and not self.type_constraint:
             ## add lexicon features
             lex_features = np.array([self.get_lex_repr(word) for word in words])
         # char embeddings
         if self.c_in_dim > 0:
             cfeatures = [self.get_c_repr(word, train=train) for word in words]
+
             if len(lex_features) > 0:
                 lex_features = dynet.inputTensor(lex_features)
                 features = [dynet.concatenate([w,c,l]) for w,c,l in zip(wfeatures,cfeatures,lex_features)]
             else:
                 features = [dynet.concatenate([w, c]) for w, c in zip(wfeatures, cfeatures)]
+            if embeds_in_file:
+                if len(lex_features) > 0:
+                    print("both contextualized embeddings and lexical features are not yet supported")
```

Comment:
How about: "Using both contextualized embeddings and lexical features simultaneously is not yet supported"?
This is ambiguous ;)

Reply:
good point, this is no longer the case, we can drop this.

Reply:
removed this. 

Now --embeds_in_file_dim is only needed during training.